src/Organizer/Serializers/UpdateContactPointDenormalizer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Organizer\Serializers;
+
+use CultuurNet\UDB3\Model\ValueObject\Contact\ContactPoint;
+use CultuurNet\UDB3\Model\ValueObject\Contact\TelephoneNumber;
+use CultuurNet\UDB3\Model\ValueObject\Contact\TelephoneNumbers;
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddresses;
+use CultuurNet\UDB3\Model\ValueObject\Web\Url;
+use CultuurNet\UDB3\Model\ValueObject\Web\Urls;
+use CultuurNet\UDB3\Organizer\Commands\UpdateContactPoint;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+final class UpdateContactPointDenormalizer implements DenormalizerInterface
+{
+    private string $organizerId;
+
+    public function __construct(string $organizerId)
+    {
+        $this->organizerId = $organizerId;
+    }
+
+    public function denormalize($data, $type, $format = null, array $context = []): UpdateContactPoint
+    {
+        $telephoneNumbers = [];
+        if (isset($data['phone'])) {
+            foreach ($data['phone'] as $phone) {
```

Comment:
Nit: I think you can remove the `if (isset(...))` by doing this:

```
foreach ($data['phone'] ?? [] as $phone) {
  // ...
}
```

Reply:
Or use `array_map` so you don't need the `$telephoneNumbers = [];` and make it a one-liner.

For example:

```
$telephoneNumbers = array_map(fn ($phone) => new TelephoneNumber($phone), $data['phone'] ?? []);
```

Reply:
The `array_map` is indeed much more elegant.